tests/test_playbooks/vars/hostgroup.yml

```diff
@@ -1,6 +1,8 @@
 ---
+foreman_proxy: foreman.example.com
```

Comment:
this should be set in server.yml and if needed, a corresponding match exception added to vcr_wrapper